django/api/models/go_electric_rebate_application.py

```diff
@@ -93,6 +94,8 @@ class Status(TextChoices):
         null=True,
         validators=[validate_file_size, validate_file_safe],
     )
+    submission_date = DateTimeField(auto_now_add=True, blank=True)
```

Comment:
Hi Vibhi, since the GoElectricRebateApplication model extends TimeStampedModel, we actually already have created and modified fields; if we wanted, can we just display those instead in the admin panel? Also, looking at the card, it seems like we want a field that tracks the approved/not-approved date; the modified date may not always be equal to that date (e.g. an application is approved on, Jan 1, then it is redeemed on Jan 7, and so the modified date will show Jan 7 instead of Jan 1 (the approval date))